pkg/model/current_backup.go

```diff
@@ -2,73 +2,28 @@ package model
 
 import (
 	"time"
-
-	"github.com/aerospike/backup-go"
 )
 
 // CurrentBackups represent the current state of backups (full and incremental)
 type CurrentBackups struct {
 	// Full represents the state of a full backup. Nil if no full backup is running.
-	Full *CurrentBackup `json:"full,omitempty"`
+	Full *RunningJob `json:"full,omitempty"`
 	// Incremental represents the state of an incremental backup. Nil if no incremental backup is running.
-	Incremental *CurrentBackup `json:"incremental,omitempty"`
+	Incremental *RunningJob `json:"incremental,omitempty"`
 }
 
-type CurrentBackup struct {
-	// TotalRecords: the total number of records to be backed up.
+// RunningJob tracks progress of currently running job.
+// @Description RunningJob tracks progress of currently running job.
+type RunningJob struct {
+	// TotalRecords: the total number of records to be processed.
 	TotalRecords uint64 `json:"total-records,omitempty" example:"100"`
-	// DoneRecords: the number of records that have been successfully backed up.
+	// DoneRecords: the number of records that have been successfully done.
 	DoneRecords uint64 `json:"done-records,omitempty" example:"50"`
 	// StartTime: the time when the backup operation started.
 	StartTime time.Time `json:"start-time,omitempty" example:"2006-01-02T15:04:05Z07:00"`
 	// PercentageDone: the progress of the backup operation as a percentage.
-	PercentageDone int `json:"percentage-done,omitempty" example:"50"`
+	PercentageDone uint `json:"percentage-done,omitempty" example:"50"`
```

Comment:
Don't we want to support percent fractions (two digits) using a float32?

Reply:
I think, integer percent is enough, why bother with floats.